src/frontend/main.ts

```diff
@@ -480,24 +478,24 @@ const vueApp = createApp(defineComponent({
         this.devicePixelRatio = this.getDevicePixelRatio();
     },
     methods: {
-        async login(ev: MouseEvent)
+        async login(username: string, password: string, areaId: string, characterId: string)
         {
             try {
                 // Workaround for making TTS work on iphone, since it needs to be activated on a user interaction
                 if (window.speechSynthesis)
                     speechSynthesis.speak(new SpeechSynthesisUtterance(""));
 
-                ev.preventDefault();
```

Comment:
I forgot why this `ev.preventDefault()` is here... It was added with cbb678fa9529e4d0ef1b06689ffa3966196f2aa1 but unfortunately there is no comment.

It was probably there for a reason, but after testing a bit everything seems fine even without this line, so I guess we can remove it for now.